osbs/conf.py

```diff
@@ -554,3 +565,23 @@ def get_platform_node_selector(self, platform):
                 nodeselector = dict([k.strip(), v.strip()] for (k, v) in raw_nodeselector.items())
 
         return nodeselector
+
+    def get_platform_descriptors(self):
+        has_v1 = False
+        platform_descriptors = {}
+        for section in self.scp.sections():
+            if "platform:" not in section:
+                continue
+            platform = section.split("platform:")[1]
+            platform_descriptor = {}
+            arch = self._get_value("architecture", section, "architecture") or platform
+            enable_v1 = self._get_value("enable_v1", section, "enable_v1", False, True)
+            if enable_v1:
+                if has_v1:
+                    raise OsbsValidationException('two platforms have enable_v1 as true')
```

Comment:
This method is pretty straight forward. A minor enhancement would be to report the multiple platforms that have requested v1. How about turning `has_v1` into a list, and add all the platforms that have requested v1 to it. Then after the for-loop executes, you can simply check the length, if it's greater than 1, an error can be raised indicating exactly which platforms are involved. This also has the benefit of checking if there's possibly a third platform that need to be dealt with.